### ISSUE TITLE
JavaScript Course:  add a resource for 'Asynchronous JavaScript and APIs' section 

### ISSUE THREAD

**Issue by Alonis8729** [TheOdinProject/curriculum#26617]
### Describe your suggestion

I found the entire  Asynchronous JavaScript and APIs section very consfusing, and the resources based on 'You don't know JavaScript' were exhausting and hard to follow.
I would like to suggest the resource:[video](https://www.youtube.com/watch?v=VmQ6dHvnKIM&t=2700s) for promises, callback functions and await/async.

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-asynchronous-code

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

**Comment by CouchofTomato** [comment]
If you want to make a PR with where you think this should go we would be happy to review it. Based on the comments in the video it seems most people think very highly of it.

**Comment by wise-king-sullyman** [comment]
@Alonso8729 are you planning to make a PR related to this issue?

It's quite a long video so I would hesitate to make it an assignment, and we're already at the maximum number of additional resources for the async lesson, but if you have reviewed all of the existing additional resources and feel it is better than the existing videos we can review switching them out.

Alternatively, would it make more sense in the async/await lesson since those seem to be featured in this video as well?
